rs/interfaces/state_manager/src/lib.rs

```diff
@@ -410,4 +412,7 @@ pub trait StateReader: Send + Sync {
     fn get_certified_state_snapshot(
         &self,
     ) -> Option<Box<dyn CertifiedStateSnapshot<State = Self::State> + 'static>>;
+
+    /// Returns a map of scheduler priorities.
+    fn get_latest_scheduler_priorities(&self) -> HashMap<CanisterId, AccumulatedPriority>;
```

Comment:
I'm not a big fan of making the state reader interface aware of this detail. I think we already have something to obtain the latest state. So we could call this and then obtain the priorities from the state we get back.

Reply:
Done.